src/CMakeLists.txt

```diff
@@ -272,7 +272,8 @@ ADD_DEFINITIONS(-DEIGEN_MPL2_ONLY)
 # - external if CERES_DIR_HINTS and FIND_PACKAGE return a valid Ceres setup
 # ==============================================================================
 MESSAGE(STATUS "Trying to find package Ceres for openMVG: ${CERES_DIR_HINTS}")
-FIND_PACKAGE(Ceres QUIET HINTS ${CERES_DIR_HINTS})
+FIND_PACKAGE(Ceres QUIET HINTS ${CERES_DIR_HINTS}
+                   REQUIRED SuiteSparse)
```

Comment:
it does not work for me. `REQUIRED` may  just overrule `QUITE` but it does not means that it only works if ceres is compiled with SuiteSparse. Indeed I tested on a ceres version without SS and it found correctly ceres without complaining.